bool objExist
int16 objPosX
int16 objPosY
